src/main/java/christmas/domain/user/order/OrderDetail.java

```diff
@@ -0,0 +1,35 @@
+package christmas.domain.user.order;
+
+import christmas.util.StringConvertor;
+
+public class OrderDetail {
+    private final OrderMenuName menuName;
+    private final Quantity quantity;
+    private final OrderPrice price;
+
+    public OrderDetail(OrderMenuName menuName, Quantity quantity, OrderPrice price) {
+        this.menuName = menuName;
+        this.quantity = quantity;
+        this.price = price;
+    }
+
+    public static OrderDetail from(String orderMenu) {
+        String[] orderInfo = StringConvertor.splitByHyphen(orderMenu);
+        OrderMenuName menuName = OrderMenuName.from(orderInfo);
+        Quantity quantity = Quantity.from(orderInfo);
```

Comment:
```suggestion
        OrderMenuName menuName = OrderMenuName.from(orderInfo[0]);
        Quantity quantity = Quantity.from(orderInfo[1]);
```
이 부분은 각 menuName과 quantity에 orderInfo 통째로 줘서 그 안에서 값을 가져오게하는 것 보다는
이렇게 풀어서 넣어주는 것은 어떨까요?
메뉴 이름이나 수량이 바로 상위 객체인 OrderDetail에 의존하고 있는데, 
더 상위 객체인 orderInfo를 의존해야 할까? 라는 생각이 들어서요!

Reply:
더불어 0, 1과 같은 매직 넘버도 메뉴이름과  수량을 가리키는 인덱스라는 표현으로 상수화 시킬 수 있을 것 같습니다!

Reply:
처음에는 인덱스 값으로 orderInfo[0] 이렇게 넘기려했는데 어떤 값을 넘기는지 알기 어렵다 생각해서
통으로 넘겨 해당 클래스의 팩토리 메서드에서 변환하여 생성하도록 했는데
인덱스를 상수화 시킨다면 쉽게 파악할 수 있겠군요 ㅎㅎ 좋은 아이디어 감사합니다!